osu.Game.Tests/Visual/Collections/TestSceneManageCollectionsDialog.cs

```diff
@@ -22,44 +22,35 @@ namespace osu.Game.Tests.Visual.Collections
 {
     public class TestSceneManageCollectionsDialog : OsuManualInputManagerTestScene
     {
-        protected override Container<Drawable> Content => content;
+        protected override Container<Drawable> Content { get; } = new Container { RelativeSizeAxes = Axes.Both };
 
-        private readonly Container content;
-        private readonly DialogOverlay dialogOverlay;
-        private readonly CollectionManager manager;
+        private DialogOverlay dialogOverlay;
+        private CollectionManager manager;
 
         private RulesetStore rulesets;
         private BeatmapManager beatmapManager;
 
         private ManageCollectionsDialog dialog;
 
-        public TestSceneManageCollectionsDialog()
+        [BackgroundDependencyLoader]
+        private void load(GameHost host)
```

Comment:
You actually want to cache before adding children, otherwise children will be loaded first and thus inject the game-wide BeatmapManager.
This is the case for both tests.

Reply:
Weird, I thought I did test this when making changes.

It turns out the usage of `content` both of these classes is scheduled, so this change isn't required. Think I should still do it for safety?

Reply:
It should be an issue in this class itself, because `CollectionManager` resolves in `BeatmapManager` and is added in `load()`.

Reply:
oh, beatmapmanager. i guess that's not visible in the tests; i thought you were talking about collectionmanager's caching.